dali/kernels/transpose/transpose_gpu_impl_test.cu

```diff
@@ -254,10 +318,9 @@ TEST(TransposeGeneric, AllPerm4D) {
   ASSERT_LT(grid_size * block_size, size) << "Weak test error: Grid too large to test grid loop";
 
   for (auto &perm : testing::Permutations4) {
-    std::cerr << "Testing permutation "
-      << perm[0] << " " << perm[1] << " " << perm[2] << " " << perm[3] << "  input shape "
-      << shape << "\n";
-      CUDA_CALL(cudaMemset(out_gpu, 0xff, size*sizeof(int)));
+    std::cerr << "Testing permutation " << perm[0] << " " << perm[1] << " " << perm[2] << " "
+              << perm[3] << "  input shape " << shape << "\n";
```

Comment:
The arguments here were carefully grouped. Autoformatter obliterated it.

Reply:
reverted